src/main/java/Duke.java

```diff
@@ -1,10 +1,144 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MARK_TASK_NUMBER_INDEX = 5;
+    public static final int UNMARK_TASK_NUMBER_INDEX = 7;
+
+    public static String markTask(String input) {
+        int itemIndex = Integer.parseInt(input.substring(MARK_TASK_NUMBER_INDEX)) - 1;
+        Task.tasks[itemIndex].setTaskStatus(true);
+        return "\tNice! I've marked this task as done:" + System.lineSeparator() + "\t\t"
+                + Task.tasks[itemIndex];
+    }
+
+    public static String unmarkTask(String input) {
+        int itemIndex = Integer.parseInt(input.substring(UNMARK_TASK_NUMBER_INDEX)) - 1;
+        Task.tasks[itemIndex].setTaskStatus(false);
+        return "\tOK, I've marked this task as not done yet:" + System.lineSeparator() + "\t\t"
+                + Task.tasks[itemIndex];
+    }
+
+    public static String getList() {
+        String output = "\tHere are the tasks in your list:" + System.lineSeparator();
+        for (int i = 0; i < Task.numOfTasks; i++) {
+            output += "\t" + (i + 1) + "." + Task.tasks[i];
+            if (i != Task.numOfTasks - 1) {
+                output += System.lineSeparator();
+            }
+        }
+        return output;
+    }
+
+    public static void createTodoTask(String input) {
+        Task.tasks[Task.numOfTasks] = new Todo(input);
+    }
+
+    public static void createDeadlineTask(String input) {
```

Comment:
I would recommend adding a Javadoc comment in front of your methods to clarify what they do and what parameters they take.